src/Backend/Modules/Settings/Layout/Templates/Index.html.twig

```diff
@@ -207,70 +219,86 @@
   </div>
   <div class="row fork-module-content">
     <div class="col-md-12">
-      <div class="panel panel-default">
-        <div class="panel-heading">
-          <h2 class="panel-title">{{ 'lbl.CkFinder'|trans|ucfirst }}</h2>
+      <div class="card card-default">
+        <div class="card-heading">
+          <h2 class="card-title">{{ 'lbl.CkFinder'|trans|ucfirst }}</h2>
         </div>
-        <div class="panel-body">
-          <div class="form-group">
-            <label for="ckfinderLicenseName" class="control-label">{{ 'lbl.LicenseName'|trans|ucfirst }}</label>
-            {% form_field ckfinder_license_name %} {% form_field_error ckfinder_license_name %}
-          </div>
-          <div class="form-group">
-            <label for="ckfinderLicenseKey" class="control-label">{{ 'lbl.LicenseKey'|trans|ucfirst }}</label>
-            {% form_field ckfinder_license_key %} {% form_field_error ckfinder_license_key %}
-          </div>
-          <div class="form-group">
-            <label for="ckfinderImageMaxWidth" class="control-label">{{ 'lbl.MaximumWidth'|trans|ucfirst }}</label>
-            {% form_field ckfinder_image_max_width %} {% form_field_error ckfinder_image_max_width %}
-            <span class="help-block">{{ 'msg.HelpCkfinderMaximumWidth'|trans|ucfirst }}</span>
-          </div>
-          <div class="form-group last">
-            <label for="ckfinderImageMaxHeight" class="control-label">{{ 'lbl.MaximumHeight'|trans|ucfirst }}</label>
-            {% form_field ckfinder_image_max_height %} {% form_field_error ckfinder_image_max_height %}
-            <span class="help-block">{{ 'msg.HelpCkfinderMaximumHeight'|trans|ucfirst }}</span>
+        <div class="card-body">
+          <div class="row">
+            <div class="col-lg-6">
+              <div class="form-group">
+                <label for="ckfinderLicenseName" class="control-label">{{ 'lbl.LicenseName'|trans|ucfirst }}</label>
+                {% form_field ckfinder_license_name %} {% form_field_error ckfinder_license_name %}
+              </div>
+            </div>
+            <div class="col-lg-6">
+              <div class="form-group">
+                <label for="ckfinderLicenseKey" class="control-label">{{ 'lbl.LicenseKey'|trans|ucfirst }}</label>
+                {% form_field ckfinder_license_key %} {% form_field_error ckfinder_license_key %}
+              </div>
+            </div>
+            <div class="col-lg-6">
+              <div class="form-group last">
```

Comment:
`:last-of-type` is hard here, because the `form-group` aren't siblings. The cols are siblings but there are a lot of different types of col classes (with the breakpoints and the col sizes). The only other option that i see is to use `mb-0` instead of `last`, because thats what the `last` class does, remove the margin